lib/Driver/DarwinToolChains.cpp

```diff
@@ -595,3 +584,13 @@ bool toolchains::Darwin::shouldStoreInvocationInDebugInfo() const {
     return S[0] != '\0';
   return false;
 }
+
+void 
+toolchains::Darwin::validateArguments(DiagnosticEngine &diags,
+                                      const llvm::opt::ArgList &args) const {
+    
+    // Validating darwin unsupported -static-stdlib argument.
```

Comment:
…and you still have a double indent here. (And a blank line that's not really needed anymore!)

Reply:
Sure, sorry for that one :))